### PR TITLE
Design system: Views and Search

### PR DESCRIPTION
**Pull request by tobyprivett**
This PR covers the views and search functionality for the back office.
It doesn't cover the forms, which will be done in the next PR

Implement Design System boilerplate
https://eaflood.atlassian.net/browse/RUBY-1457

Styling: Dashboard
https://eaflood.atlassian.net/browse/RUBY-1573

Styling: View registration
https://eaflood.atlassian.net/browse/RUBY-1494

Styling: View users
https://eaflood.atlassian.net/browse/RUBY-1502

Refactor: Search
https://eaflood.atlassian.net/browse/RUBY-1477

### CODE REVIEW

**Comment by tobyprivett** [comment]
@irisfaraway Can we squash this Sonarcloud issue? https://sonarcloud.io/project/issues?id=DEFRA_flood-risk-back-office&issues=AXuIUDv8ucP33EKIDV_l&open=AXuIUDv8ucP33EKIDV_l&pullRequest=312

File: app/views/layouts/flood_risk_engine.html.erb [hunk 1]
```diff
@@ -28,4 +28,4 @@
   <%= content_tag :div, yield %>
 <% end -%>
 
-<%= render template: 'layouts/govuk_admin_template' %>
+<%#= render template: 'layouts/govuk_admin_template' %>
```
Comment (irisfaraway) [inline app/views/layouts/flood_risk_engine.html.erb:31]: Should we delete this entirely?

File: app/views/admin/enrollment_exemptions/show/_correspondant.html.erb [hunk 2]
```diff
@@ -1,51 +1,37 @@
...
-      </div>
-    </div>
-  </div>
+<div id="correspondence-contact-details" class="govuk-!-padding-bottom-2">
+  <h2 class="govuk-heading-m"><%= t ".contact_header" %></h2>
+
+  <% contact = presenter.correspondence_contact %>
+  <p><%= contact.full_name %></p>
+
+  <% if !contact.position? %>
```
Comment (irisfaraway) [inline app/views/admin/enrollment_exemptions/show/_correspondant.html.erb:7]: Should this be `contact.position?` instead? Looks like it's changed but not sure why

**Review by irisfaraway — COMMENTED** [review]
Just some minor things!

Looks like SonarCloud is requesting some more code coverage too – are there any additional bits we can test? I did resolve the issue with the `<li>` tags.

File: app/views/layouts/flood_risk_engine.html.erb (hunk 1 above)
Comment (tobyprivett) [reply app/views/layouts/flood_risk_engine.html.erb:31]: Yes, it's to render the old template, which was installed with the 'govuk_admin_template' gem, but is no longer there.

File: app/views/admin/enrollment_exemptions/show/_correspondant.html.erb (hunk 2 above)
Comment (tobyprivett) [reply app/views/admin/enrollment_exemptions/show/_correspondant.html.erb:7]: Well spotted! My debugging code.

**Comment by tobyprivett** [comment]
> Just some minor things!
> 
> Looks like SonarCloud is requesting some more code coverage too – are there any additional bits we can test? I did resolve the issue with the `<li>` tags.

This PR is all templating (except for the search) so I'm not sure how to go about backwards-testing. Unless we find some bugs?

File: app/views/layouts/flood_risk_engine.html.erb (hunk 1 above)
Comment (tobyprivett) [reply app/views/layouts/flood_risk_engine.html.erb:31]: On closer inspection - this layout renders the engine / journey pages. We have a ticket to integrate this, so I think we should leave the file as is, for now.

File: app/views/admin/enrollment_exemptions/show/_correspondant.html.erb (hunk 2 above)
Comment (tobyprivett) [reply app/views/admin/enrollment_exemptions/show/_correspondant.html.erb]: Fixed in: https://github.com/DEFRA/flood-risk-back-office/pull/312/commits/1a9e427fbf00e009ae85d666259fbaa72c0b0e13

**Comment by irisfaraway** [comment]
@tobyprivett It looks like the test coverage complaints are about the search: https://sonarcloud.io/component_measures?id=DEFRA_flood-risk-back-office&metric=new_lines_to_cover&pullRequest=312&selected=DEFRA_flood-risk-back-office%3Aapp%2Fqueries%2Fenrollment_exemption_search_query.rb&view=list

So if we can add some more tests for that, that would be great :)

**Review by irisfaraway — APPROVED** [review]

**Pull request merged**